configs/config.yaml

```diff
@@ -1,10 +1,13 @@
 eth:
-  protocol: ws
-  host: geth
-  port: 8546
+  - protocol: ws
```

Comment:
How about make flags and configs consistent? Pass array like:
eth:
  - ws://127.0.0.1:8585
  - ws://127.0.0.1:8586